### PR TITLE
fix warnings

### LINKED ISSUE #5117: Doc warnings
## Issue Details
There are warnings in our doc :
 - /home/cgal-testsuite/cgal_doc_build/CGAL-5.2-I-34/doc/BGL/Concepts/FaceGraph.h:28: warning: explicit link request to 'face_descriptor' could not be resolved
 - /home/cgal-testsuite/cgal_doc_build/CGAL-5.2-I-34/doc/BGL/Concepts/HalfedgeGraph.h:47: warning: explicit link request to 'halfedge_descriptor' could not be resolved
 - /home/cgal-testsuite/cgal_doc_build/CGAL-5.2-I-34/include/CGAL/Variational_shape_approximation.h:1832: warning: end of comment block while expecting command </tt>
## Environment

* CGAL version:5.0

### PR DESCRIPTION
**Pull request by sloriot**
fixes #5117

### CODE REVIEW

File: Surface_mesh_approximation/include/CGAL/Variational_shape_approximation.h [hunk 1]
```diff
@@ -1778,7 +1778,7 @@ class Variational_shape_approximation {
   }
 
   /*!
-   * @brief subdivides a chord recursively in range `[@a chord_begin, @a chord_end).`
+   * @brief subdivides a chord recursively in range `[chord_begin, chord_end).`
    * @param chord_begin begin iterator of the chord
    * @param chord_end end iterator of the chord
```
Comment (albert-github) [inline Surface_mesh_approximation/include/CGAL/Variational_shape_approximation.h:1783]: This problem is already addressed in #5102 and in #5120 there is an essential part of the fix missing i.e. bringing the `.` outside of the backtics.

Comment (sloriot) [reply Surface_mesh_approximation/include/CGAL/Variational_shape_approximation.h:1783]: I knew I saw that fix already :)

File: Surface_mesh_approximation/include/CGAL/Variational_shape_approximation.h [hunk 2]
```diff
@@ -1778,7 +1778,7 @@ class Variational_shape_approximation {
   }
 
   /*!
-   * @brief subdivides a chord recursively in range `[@a chord_begin, @a chord_end).`
+   * @brief subdivides a chord recursively in range `[chord_begin, chord_end).`
```
Comment (sloriot) [inline Surface_mesh_approximation/include/CGAL/Variational_shape_approximation.h:1781]:
```suggestion
   * @brief subdivides a chord recursively in range `[chord_begin, chord_end)`.
```

File: Surface_mesh_approximation/include/CGAL/Variational_shape_approximation.h (hunk 1 above)
Comment (albert-github) [reply Surface_mesh_approximation/include/CGAL/Variational_shape_approximation.h]: OK I'll close my proposed pull request #5102 in favor of this pull request

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.2-Ic-36.shtml

**Pull request merged**